requirement.txt

```diff
@@ -0,0 +1,6 @@
+click==7.1.2
+Flask==1.1.2
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+Werkzeug==1.0.1
```

Comment:
Add the following requirements also:
Python >= v2.7.0  || v3.7.0
Node.js >= v0.10.0
Pip >= v9.0.1

Python packages: flask

Reply:
@Kashish121  could you tell me why this is required as pip and python is the outside dependencies means firstly then user will activate the virutalenv and then hit the python commands. @vinitshahdeo  Let me know you thought also.